### PR TITLE
[TT-5508] Update with Postman service

### PR DESCRIPTION
**Pull request by marksou**

### CODE REVIEW

File: tyk-docs/content/advanced-configuration/websockets.md [hunk 1]
```diff
@@ -24,24 +24,29 @@ https://target:443/
...
+Set the **Target URL** to `wss://ws.postman-echo.com/raw`
 
-This gives you a **Gateway URL** of: `http://tyk-test.com:8080/websocket/`
-
-We will change `http` to `ws` in the websocket.org test site.
 
 ### Step 2. Test the Connection
 
-From http://websocket.org/echo.html enter the following in the **Location** field.
+1. From Postman, select **File > New > WebSocket Request**.
```
Comment (pagemag) [inline tyk-docs/content/advanced-configuration/websockets.md]: It should not be `File > New > WebSocket Request` but `Choose a Workspace > New > WebSocket Request`

File: tyk-docs/content/advanced-configuration/websockets.md [hunk 2]
```diff
@@ -24,24 +24,29 @@ https://target:443/
...
-`ws://www.tyk-test.com:8080/websocket/`
+{{< img src="/img/dashboard/system-management/postman-websocket-test.png" alt="Postman WebSocket Connection Result" >}}
 
-Enter some text in the **Message** field, and click **Send**. You should see your sent message in the **Log** field.
+{{< note success >}}
+**Note**  
 
-![WebSocket Test](/docs/img/dashboard/system-management/websocket_test.png)
\ No newline at end of file
+If your API uses an Authentication mode other than Open (Keyless), add the details in the Header tab.
```
Comment (pagemag) [inline tyk-docs/content/advanced-configuration/websockets.md:51]: Can we add an image of a sample on how to achieve this as there are customers asking on how to use headers when using websocket.

**Review by pagemag — CHANGES_REQUESTED** [review]
Hi @marksou, please see some of my reviews. Thank you!

File: tyk-docs/content/advanced-configuration/websockets.md (hunk 1 above)
Comment (marksou) [reply tyk-docs/content/advanced-configuration/websockets.md]: @pagemag It's available from both. Which I admit is a bit strange!

Comment (pagemag) [reply tyk-docs/content/advanced-configuration/websockets.md]: Oh, I just saw that but I checked on the web version, there's no File.

Comment (marksou) [reply tyk-docs/content/advanced-configuration/websockets.md]: Ok, I'll add both methods :-)

File: tyk-docs/content/advanced-configuration/websockets.md (hunk 2 above)
Comment (marksou) [reply tyk-docs/content/advanced-configuration/websockets.md:51]: Added in latest commit.

**Review by pagemag — APPROVED** [review]
Thanks Mark! Approving the changes.

**Pull request merged**